packages/app/package.json

```
@@ -70,6 +73,9 @@
       "compileSdk": 30,
       "buildTools": "30.0.2",
       "firebase": "28.2.1",
+      "firebaseCrashlytics": "2.7.1",
+      "firebasePerf": "1.4.0",
+      "gmsGoogleServices": "4.3.8",
```

With apologies, as this is a nitpick, but these strings are ambiguous to me, firebase crashlytics may refer to multiple things (as might the other three) so the name could use disambiguation - this impact all code referencing them

```suggestion
      "firebaseCrashlyticsPlugin": "2.7.1",
      "firebasePerfPlugin": "1.4.0",
      "gmsGoogleServicesPlugin": "4.3.8",
```

---

These versions directly correspond to gradle dependencies from RNFB installation docs: 
- https://rnfirebase.io/#configure-firebase-with-android-credentials - `gmsGoogleServices`
- https://rnfirebase.io/crashlytics/android-setup#2-add-the-firebase-crashlytics-plugin-dependency - crashlytics
- https://rnfirebase.io/perf/usage#add-the-performance-monitoring-plugin - perf

Instead of `Plugin` suffix, I'd prefer more accurate suffix - `Gradle`, e.g. `firebaseCrashlyticsGradle`, unless you have any better idea. As @Salakar mentioned, these versions can be used for automatic docs generation.

---

Yeah - the versions are ace, it's just disambiguating the variable name as it will escape it's gradle-local context here now that it's in package.json. I like your idea of a gradle suffix, certainly satisfies the issue I had in mind